app/build.gradle

```diff
@@ -10,6 +10,8 @@ apply plugin: 'androidx.navigation.safeargs.kotlin'
 
 def STRIPE_API_TOKEN = System.getenv('STRIPE_API_TOKEN') ?: "YOUR_API_KEY"
 def MAPBOX_KEY = System.getenv('MAPBOX_KEY') ?: "pk.eyJ1IjoiYW5nbWFzMSIsImEiOiJjanNqZDd0N2YxN2Q5NDNuNTBiaGt6eHZqIn0.BCrxjW6rP_OuOuGtbhVEQg"
+def FB_APP_ID = System.getenv('FB_APP_ID') ?: "NOT_PROVIDED"
+def FB_LOGIN_PROTOCOL = System.getenv("FB_LOGIN_PROTOCOL") ?: "NOT_PROVIDED"
```

Comment:
What should be passed here?

Reply:
I have declared these as environmental can be set during release build, same as strip key.